app-modules/etno/src/Http/Resources/ItemResource.php

```
@@ -82,6 +82,8 @@ public function toArray(Request $request): array
             'keywords' => KeywordResource::collection($this->whenLoaded('keywords')),
             'research_collections' => ResearchCollectionResource::collection($this->whenLoaded('researchCollections')),
             'document_id' => $this->document_id,
+            /** @var MediaType|null */
+            'media_type' => $this->whenLoaded('firstMedia', fn () => MediaType::tryFrom($this->firstMedia?->collection_name)),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Metafori\Etno\Enums\MediaType' in method 'toArray'. (undefined)

(StaticAccess)